recipes/turbo_turtle/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "turbo_turtle" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lanl-aea/turbo-turtle/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e89f218c8d95484f3b0b8b61427c0bd82779226301436c991535966bac4d140d
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  entry_points:
+    - turbo-turtle = turbo_turtle._main:main
+  ignore_run_exports:
+    - python_abi
+
+requirements:
+  build:
+    - lazy_loader >=0.4
+    - scons >=4
+    - setuptools_scm[toml] >=6.2
```

Comment:
Things in square brackets are basically ignored. Thia doesn't do what you expect.

Reply:
This used to be the recommended requirement from the setuptools_scm package. Poking around a little deeper, it looks like this was always a PIP syntax and never a Conda syntax.

It looks like they've updated their documentation: https://setuptools-scm.readthedocs.io/en/stable/#with-setuptools. I can update the recipe and upstream package to reflect the current docs.